test/Microsoft.AspNet.Mvc.Core.Test/ParameterBinding/ModelBindingHelperTest.cs

```diff
@@ -23,10 +23,7 @@ public class ModelBindingHelperTest
         public async Task TryUpdateModel_ReturnsFalse_IfBinderReturnsFalse()
         {
             // Arrange
-            var metadataProvider = new Mock<IModelMetadataProvider>();
-            metadataProvider.Setup(m => m.GetMetadataForType(It.IsAny<Type>()))
-                            .Returns(new ModelMetadata(metadataProvider.Object, null, typeof(MyModel), null))
-                            .Verifiable();
+            var metadataProvider = new EmptyModelMetadataProvider();
```

Comment:
When should I be using an ```EmptyModelMetadataProvider``` vs TestModelMetadataProvider.CreateDefaultProvider() ?